### DISCUSSION TITLE
APOE: error about boolean dimension

### DISCUSSION THREAD

**Discussion by krootesmurdy** [trendscenter/coinstac#1543] [General]
error running regression (voxel size set at 1.5)

IndexError: boolean index did not match indexed array along dimension 0; dimension is 145 but corresponding boolean dimension is 109

solution: increasing the voxel size. however, the inputted image is 1.5mm so I'm not sure what this input is asking for? or if we even need it?

**Comment by hvgazula** [comment]
@krootesmurdy  What's the voxel size of the MNI template?

**Comment by spanta28** [comment]
I thought we faced this error before when covariates and number of datasets
did not match.

**Reply by hvgazula** [reply]
This has to do with the resampling.

**Comment by spanta28** [comment]
Ok
